back/node_watcher/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:alpine3.10
+
+ENV PRISMA_ENDPOINT http://prisma:4466
```

Comment:
did you mean `http://localhost:4466`, or does prisma here actually refer to the image?

Reply:
Yes,in the virtualnetwork docker-compose creates, to access a container you put its image name

Reply:
ah right 👍